<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1255</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>660</y>
     <width>1221</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="BottomLayout"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>20</y>
     <width>501</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="MoveLayout"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>440</y>
     <width>501</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="setAndHealthLayout"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>230</y>
     <width>501</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="pidLayout"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>721</width>
     <height>641</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="oscilloScopeVerticalLayout"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
